ptf/sailag.py

```diff
@@ -53,7 +48,8 @@ def createRemoveLagMemberTest(self):
         # verify LAG and LAG members
         attr_list = sai_thrift_get_lag_attribute(
             self.client, lag3, port_list=portlist)
-        # TODO: lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        print(lag_members)
```

Comment:
Did you verify it works correctly? If so, and the lag_members list is retrieved, we can also verify if the list contains all expected lag members.
\+ this print should be removed